modules/URL/ConferenceUrl.js

```diff
@@ -67,7 +67,14 @@ export default class ConferenceUrl {
      * Reloads the conference using original URL with all of the parameters.
      */
     reload() {
-        logger.info("Reloading the conference using URL: " + this.originalURL);
-        replace(this.originalURL);
+        // Check if we are in iframe and reload with the reload() utility
+        // because replace() is not working on iframe.
+        if(window.self !== window.top) {
+            reload();
+        } else {
+            logger.info("Reloading the conference using URL: "
```

Comment:
don't we want the log line always? Also, maybe switch to a template string since we're here.